src/index.jade

```diff
@@ -24,7 +24,19 @@ html(lang="en")
     link(href="/css/vendor.css", rel="stylesheet")
     link(href="/css/all.css", rel="stylesheet")
     base(href="/")
-  body(ng-app="wisboo")
+  body(ng-app="librarium")
+    header
+      nav.navbar.navbar-default
+        .container
+          .navbar-header
+            a(href="/").navbar-brand Librarium
```

Comment:
Our convention is to put the classname in front of the properties, like this
`a.navbar-brand(href="/")`
Also, Text should be in different line preceded by | like this:
`|   Librarium`